bus-mapping/src/evm/opcodes/ids.rs

```diff
@@ -0,0 +1,408 @@
+    pub const PUSH11: OpcodeId = OpcodeId(0x6a);
+    pub const PUSH12: OpcodeId = OpcodeId(0x6b);
+    pub const PUSH13: OpcodeId = OpcodeId(0x6c);
+    pub const PUSH14: OpcodeId = OpcodeId(0x6d);
+    pub const PUSH15: OpcodeId = OpcodeId(0x6e);
+    pub const PUSH16: OpcodeId = OpcodeId(0x6f);
+    pub const PUSH17: OpcodeId = OpcodeId(0x70);
+    pub const PUSH18: OpcodeId = OpcodeId(0x71);
+    pub const PUSH19: OpcodeId = OpcodeId(0x72);
+    pub const PUSH20: OpcodeId = OpcodeId(0x73);
+    pub const PUSH21: OpcodeId = OpcodeId(0x74);
+    pub const PUSH22: OpcodeId = OpcodeId(0x75);
+    pub const PUSH23: OpcodeId = OpcodeId(0x76);
+    pub const PUSH24: OpcodeId = OpcodeId(0x77);
+    pub const PUSH25: OpcodeId = OpcodeId(0x78);
+    pub const PUSH26: OpcodeId = OpcodeId(0x79);
+    pub const PUSH27: OpcodeId = OpcodeId(0x7a);
+    pub const PUSH28: OpcodeId = OpcodeId(0x7b);
+    pub const PUSH29: OpcodeId = OpcodeId(0x7c);
+    pub const PUSH30: OpcodeId = OpcodeId(0x7d);
+    pub const PUSH31: OpcodeId = OpcodeId(0x7e);
+    pub const PUSH32: OpcodeId = OpcodeId(0x7f);
+
+    /// `DUPn`
+    pub const DUP1: OpcodeId = OpcodeId(0x80);
+    pub const DUP2: OpcodeId = OpcodeId(0x81);
+    pub const DUP3: OpcodeId = OpcodeId(0x82);
+    pub const DUP4: OpcodeId = OpcodeId(0x83);
+    pub const DUP5: OpcodeId = OpcodeId(0x84);
+    pub const DUP6: OpcodeId = OpcodeId(0x85);
+    pub const DUP7: OpcodeId = OpcodeId(0x86);
+    pub const DUP8: OpcodeId = OpcodeId(0x87);
+    pub const DUP9: OpcodeId = OpcodeId(0x88);
+    pub const DUP10: OpcodeId = OpcodeId(0x89);
+    pub const DUP11: OpcodeId = OpcodeId(0x8a);
+    pub const DUP12: OpcodeId = OpcodeId(0x8b);
+    pub const DUP13: OpcodeId = OpcodeId(0x8c);
+    pub const DUP14: OpcodeId = OpcodeId(0x8d);
+    pub const DUP15: OpcodeId = OpcodeId(0x8e);
+    pub const DUP16: OpcodeId = OpcodeId(0x8f);
+
+    /// `SWAPn`
+    pub const SWAP1: OpcodeId = OpcodeId(0x90);
+    pub const SWAP2: OpcodeId = OpcodeId(0x91);
+    pub const SWAP3: OpcodeId = OpcodeId(0x92);
+    pub const SWAP4: OpcodeId = OpcodeId(0x93);
+    pub const SWAP5: OpcodeId = OpcodeId(0x94);
+    pub const SWAP6: OpcodeId = OpcodeId(0x95);
+    pub const SWAP7: OpcodeId = OpcodeId(0x96);
+    pub const SWAP8: OpcodeId = OpcodeId(0x97);
+    pub const SWAP9: OpcodeId = OpcodeId(0x98);
+    pub const SWAP10: OpcodeId = OpcodeId(0x99);
+    pub const SWAP11: OpcodeId = OpcodeId(0x9a);
+    pub const SWAP12: OpcodeId = OpcodeId(0x9b);
+    pub const SWAP13: OpcodeId = OpcodeId(0x9c);
+    pub const SWAP14: OpcodeId = OpcodeId(0x9d);
+    pub const SWAP15: OpcodeId = OpcodeId(0x9e);
+    pub const SWAP16: OpcodeId = OpcodeId(0x9f);
+
+    /// `RETURN`
+    pub const RETURN: OpcodeId = OpcodeId(0xf3);
+    /// `REVERT`
+    pub const REVERT: OpcodeId = OpcodeId(0xfd);
+
+    /// `INVALID`
+    pub const INVALID: OpcodeId = OpcodeId(0xfe);
+}
+
+// External opcodes
+impl OpcodeId {
```

Comment:
`BASEFEE` is missing, it's `OpcodeId(0x48)`

Reply:
Good catch! Thanks!!